multi/nrf91-multi/nrf91-multi.h

```diff
@@ -0,0 +1,19 @@
+/*
+ * Phoenix-RTOS
+ *
+ * nRF91 multidriver
+ *
+ * Copyright 2018, 2024 Phoenix Systems
+ * Author: Aleksander Kaminski, Damian Loewnau
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#ifndef NRF91_MULTI_H_
+#define NRF91_MULTI_H_
+
+#include <sys/interrupt.h>
```

Comment:
This is an exported header (i.e. for a user to learn about driver API). No need for this include here and I guess right now no need for this header too, as the driver provides only tty, which API is standard